finetune/model.py

```diff
@@ -229,9 +254,15 @@ def _model_fn(features, labels, mode, params):
                 if mode == tf.estimator.ModeKeys.PREDICT:
                     if lm_predict_op is not None:
                         predictions[PredictMode.GENERATE_TEXT] = lm_predict_op
-                    predictions[PredictMode.LM_PERPLEXITY] = language_model_state[
-                        "perplexity"
-                    ]
+                        if lm_type.lower() == 'mlm':
+                            predictions[PredictMode.MLM_IDS] = mlm_ids
+                            predictions[PredictMode.MLM_POSITIONS] = mlm_positions
+                            #predictions[PredictMode.MLM_IDS] = features['mlm_ids']
+                            #predictions[PredictMode.MLM_POSITIONS] = features['mlm_positions']
+                    if lm_type.lower == 'lm':
```

Comment:
Indentation here is off.  This needs to be indented once more.

Reply:
indented.